frigate/http.py

```diff
@@ -1584,20 +1584,10 @@ def recordings_summary(camera_name):
         )
         .where(Recordings.camera == camera_name)
         .group_by(
-            fn.strftime(
```

Comment:
This is removing the timezone handling for recordings which is incorrect

Reply:
Why so? Grouping is done by the hour, which does not depend on timezone?

Reply:
it does depend on the timezone, the api accepts a timezone query which affects how hours are displayed. That timezone query arg is now just ignored in this implementation

Reply:
It isn't ignored, it is still in the select part of the query. Only grouping is now done based on the Unix epoch, the output of the query is exactly the same.

Reply:
I see what you're saying, this will still cause problems for timezones that have 30 minute and 15 minute offsets from UTC